src/components/Select.jsx

```diff
@@ -1,11 +1,24 @@
-import React from 'react'
+import React, { useId } from "react";
 
-const Select = () => {
+const Select = ({ options, label, className, ...props }) => {
+  const id = useId();
   return (
-    <div>
-      
+    <div className="w-full">
+      {label && <label htmlFor={id} className=""></label>}
```

Comment:
how are you deciding the styles for the components? ( for example select here )? is there any design system/mockups OR are you doing it arbitarily?

Reply:
Yes there is a design system.